contrib-04-HOP-LoKi-Functor/ProceedingHOP.md

```diff
@@ -37,22 +36,7 @@ This way we are left with the corrected HOP mass of the original $B$.
 
 # LoKi functor implementation
 
-In order to solve this task we have to make changes to the DaVinci software package. So first, we will need to set up a development environment. Use the `lb-dev` command:
-
-```bash
-lb-dev --name DaVinciDevWithHOPLoKi DaVinci v39r1p1
-```
-
-This creates `DaVinciDevWithHOPLoKi` folder, which will be our working folder. 
-
-The LoKi toolkit lives in the `Phys/LoKiPhys` package. So first we move into the `DaVinciDevWithHOPLoKi` folder and download the package from the repository.
-
-```bash
-cd ./DaVinciDevWithHOPLoKi
-getpack Phys/LoKiPhys
-```
-
-Now you will be asked about the version of package you want to download. At the time of writing of this tutorial the reliably working version was `v11r7` so choose that one. This creates a new `Phys/LoKiPhys` containing the LoKiPhys code. According to the usual LHCb convention the C++ files are located in the `src` directory and header files are in the `LoKi` directory. The various LoKi functors are logically grouped in `Particles*.{cpp,h}` files. The files where we implemented our new HOP LoKi functor is `Particles38.{cpp,h}`, since these files already contain the MCorrected (CORRM) and MCorrectedWithBestVertex (BPVCORRM) functors, which will serve as a skeleton for our new functor. MCorrectedWithBestVertex implicitly takes the best primary vertex of $B$, whereas the MCorrected uses the primary vertex that is provided as an input argument. Following this example, we also create two new LoKi functors: BremMCorrected (HOPM) and BremMCorrectedWithBestVertex (BPVCORRM).
+In order to implement the HOP mass as a LoKi functor, we have to make changes to the DaVinci software package. So first, we need to set up a development environment for the head version of `DaVinci` using the LHCb tools. The LoKi toolkit lives in the `Phys/LoKiPhys` package, the head version of which we download from the `Phys` repository. According to the usual LHCb convention, the C++ files are located in the `src` directory and header files are in the `LoKi` directory. The various LoKi functors are logically grouped in `Particles*.{cpp,h}` files. The files where we implemented our new HOP LoKi functor are `Particles38.{cpp,h}` because these files already contain the MCorrected (CORRM) and MCorrectedWithBestVertex (BPVCORRM) functors, which are to some extent similar to the functors we want to write as they aim at correcting the mass of the parent particle using the missing momentum transverse to the direction of flight. MCorrectedWithBestVertex implicitly takes the best primary vertex of $B$, whereas the MCorrected uses the primary vertex that is provided as an input argument. Following this example, we also create two new LoKi functors: BremMCorrected (HOPM) and BremMCorrectedWithBestVertex (BPVCORRM). Additionaly, the computation of $p_{\mathrm{T}}$ with respect to the $B$ flight direction is already implemented in the LoKi code and it is used as it is.
```

Comment:
"The files where we implemented", start a new paragraph before, and maybe "The files in which the new HOP LoKi functor was implemented"? The second part is because I prefer passive voice to "we", but this means correcting the whole document (although this happens in very few places)